README.md

```diff
@@ -50,6 +50,7 @@ The following sections document how to download, install, configure, use and tro
 - [General Requirements](#general-requirements)
   - [Checking the system software version](#checking-the-system-software-version)
   - [Checking MotoPlus configuration](#checking-motoplus-configuration)
+- [Option compatibility](#option-compatibility)
```

Comment:
not sure about the title of the section.

I alternated between *option*, *feature* and some other things I don't remember.

@ted-miller: would there be any Yaskawa-standard terminology we could use for this?

Reply:
`Option Function` is the official term